src/renderer/store/modules/invidious.js

```diff
@@ -21,25 +22,39 @@ const getters = {
 }
 
 const actions = {
-  async fetchInvidiousInstances({ commit }) {
+  async fetchInvidiousInstances({ commit }, payload) {
     const requestUrl = 'https://api.invidious.io/instances.json'
 
     let response
+    let instances = []
     try {
       response = await $.getJSON(requestUrl)
+      instances = response.filter((instance) => {
+        if (instance[0].includes('.onion') || instance[0].includes('.i2p')) {
+          return false
+        } else {
+          return true
+        }
+      }).map((instance) => {
+        return instance[1].uri.replace(/\/$/, '')
+      })
     } catch (err) {
       console.log(err)
-    }
-
-    const instances = response.filter((instance) => {
-      if (instance[0].includes('.onion') || instance[0].includes('.i2p')) {
-        return false
+      const fileName = 'invidious-instances.json'
+      let fileData
+      /* eslint-disable-next-line */
+      const fileLocation = payload.isDev ? './static/' : `${__dirname}/static/`
+      if (fs.existsSync(`${fileLocation}${fileName}`)) {
+        console.log('reading static file for invidious instances')
+        fileData = fs.readFileSync(`${fileLocation}${fileName}`)
```

Comment:
The new code for `instances` is specific to the expected content of the expected static file
```suggestion
        fileData = fs.readFileSync(`${fileLocation}${fileName}`)
        instances = JSON.parse(fileData).map((entry) => {
	        return entry.url
	    })
```